src/object/srv_ec_aggregate.c

```diff
@@ -1645,7 +1646,7 @@ agg_process_stripe(struct ec_agg_entry *entry)
 
 	/* Query the parity */
 	rc = vos_iterate(&iter_param, VOS_ITER_RECX, false, &anchors,
-			 agg_recx_iter_pre_cb, NULL, entry, NULL);
+			 agg_recx_iter_pre_cb, NULL, entry, dth);
```

Comment:
The return code for this (nested) invocation of vos_iterate does not currently propagate up to the outer iterator.  If this is necessary for your fix, then I can redo the return code handling.

Reply:
I refreshed the patch to resolve such issue.